models/event.js

```diff
@@ -44,14 +44,30 @@ Event.init(
       type: DataTypes.INTEGER,
       allowNull: false,
     },
-    participationJSON: {
-      type: DataTypes.STRING,
-      allowNull: false,
+    eccPoints: {
+      type: DataTypes.INTEGER,
+      allowNull: true,
     },
     description: {
       type: DataTypes.TEXT,
       allowNull: true,
     },
+    collaborator1: {
+      type: DataTypes.STRING,
+      allowNull: true,
+      references: {
+        model: Organizer,
+        key: "orgDept",
+      },
+    },
+    collaborator2: {
+      type: DataTypes.STRING,
+      allowNull: true,
+      references: {
+        model: Organizer,
+        key: "orgDept",
+      },
+    },
   },
```

Comment:
To-Do: Add a string called url to hold the url of the uploaded image for the event.

(I'm just pushing everything on you ATP :D )